app/assets/stylesheets/components/_expander.scss

```diff
@@ -30,6 +30,14 @@
   .app-c-expander__content--visible {
     display: block;
   }
+
+  [aria-expanded="true"] ~ .app-c-expander__icon--up {
```

Comment:
It seems like these were meant to be applied only when javascript is enabled (previously being nesting in `.js-enabled`) – is there a reason for not preserving this behaviour? I admit I haven't checked this locally to see if there's a logic behind it.

Reply:
They're still applied only when js is enabled: https://github.com/alphagov/finder-frontend/blob/957c8787db9ad3728b8ff7e9997ee827187b229a/app/assets/stylesheets/components/_expander.scss#L20-L41

The linter was complaining about the duplicate .js-enabled definition so I grouped them into one.

Reply:
Gocha!